x/mongo/driver/operation/aggregate.go

```diff
@@ -393,3 +397,13 @@ func (a *Aggregate) HasOutputStage(hos bool) *Aggregate {
 	a.hasOutputStage = hos
 	return a
 }
+
+// CustomOptions specifies extra options to use in the aggregate command.
+func (a *Aggregate) CustomOptions(co map[string]bsoncore.Value) *Aggregate {
```

Comment:
Same comment as above: Does it make sense to use `bson.M` here instead of `map[string]interface{}`?

Reply:
Yep. Switched.